packages/p2p/src/utils/websocket.js

```diff
@@ -72,106 +65,64 @@ const getModifiedP2POfferList = (response) => {
     return (modified_response);
 };
 
+const getModifiedP2POrder = (response) => {
+    const modified_response = {};
+
+    modified_response.offer_amount               = +response.amount;
+    modified_response.display_offer_amount       = formatMoney(response.offer_currency, response.amount);
+    modified_response.order_purchase_datetime    = new Date(response.created_time); // TOOD: [p2p-api-request] API should give epoch
+    modified_response.advertiser_notes           = response.description;
+    modified_response.order_id                   = response.id;
+    modified_response.offer_currency             = response.offer_currency;
+    modified_response.type                       = response.offer_type;
+    modified_response.status                     = response.status;
+
+    // TOOD: [p2p-api-request] missing in API
+    modified_response.advertiser_name            = response.agent_name;
+    modified_response.price_rate                 = +response.price;
+    modified_response.display_price_rate         = formatMoney(response.local_currency, response.price);
+    modified_response.transaction_currency       = response.local_currency;
+    modified_response.transaction_amount         = +response.transaction_amount;
+    modified_response.display_transaction_amount = formatMoney(response.local_currency, response.transaction_amount);
+    modified_response.remaining_time             = 60 * 60 * 1000;
```

Comment:
add a comment for how much time this is in human readable format?

Reply:
we'll get this from the API in the future, but sure I'll add a comment for now